inference-engine/src/transformations/include/ngraph_ops/rnn_cell_ie.hpp

```diff
@@ -17,6 +17,9 @@ namespace op {
 
 class TRANSFORMATIONS_API RNNCellIE : public Op {
 public:
+    static constexpr NodeTypeInfo type_info{"RNNCellIE", 1};
```

Comment:
Why don't you use `RTTI_DECLARATION` here?

Reply:
I haven't applied new RTTI to all classes. Only small fraction for illustration purposes. This one is just a preparation to consolidate RTTI-related stuff exactly at the class definition beginning. It will allow applying auto-replacement tool in later PRs.